code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -65,163 +86,341 @@
-						'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
-						'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg')
-			playsound(get_turf(src), pick(creepyasssounds), 100, 1)
+	lastsinPerson -= (sinPersonTime/4)
+	Defend(M,M)
+
+/mob/living/simple_animal/hostile/hades/proc/Defend(var/mob/attacker,var/source)
+	if(!isDoingDeath)
+		rageLevel += 5
+		src.visible_message("<span class='warning'>[src] rounds on the [attacker], gazing at them with a [pick("cold","frosty","freezing","dark")] [pick("glare","gaze","glower","stare")].</span>")
+
+		if(istype(source,/obj/item/projectile))
+			src.visible_message("<span class='warning'>[src] [pick("calmly","silently","nonchalantly")] waves their hand, deflecting the [source].</span>")
+			var/obj/item/projectile/P = source
+			if(P.starting)
+				var/new_x = P.starting.x + pick(0, 0, 0, 0, 0, -1, 1, -2, 2)
+				var/new_y = P.starting.y + pick(0, 0, 0, 0, 0, -1, 1, -2, 2)
+				var/turf/curloc = get_turf(src)
+
+				P.original = locate(new_x, new_y, P.z)
+				P.starting = curloc
+				P.current = curloc
+				P.firer = src
+				P.yo = new_y - curloc.y
+				P.xo = new_x - curloc.x
+				P.Angle = null
+		else
+			if(prob(20))
+				var/chosenDefend = rand(1,3)
+				switch(chosenDefend)
+					if(1)
+						attacker.visible_message("<span class='warning'>[attacker] is lifted from the ground, shadowy powers tossing them aside.</span>")
+						attacker.throw_at_fast(pick(orange(src,7)),10,1)
+					if(2)
+						attacker.visible_message("<span class='warning'>[attacker] crackles with electricity, a bolt leaping from [src] to them.</span>")
+						sinLightning.Bolt(src,attacker,30,5,src)
+					if(3)
+						src.visible_message("<span class='warning'>[src] points his staff at [attacker], a swarm of eyeballs lurching fourth!</span>")
+						for(var/i in 1 to 4)
+							var/mob/living/simple_animal/hostile/carp/eyeball/E = new/mob/living/simple_animal/hostile/carp/eyeball(pick(orange(attacker,1)))
+							E.faction = faction
+							spawn(150)
+								if(E)
+									E.gib()
+
+/mob/living/simple_animal/hostile/hades/proc/sinShed(var/turf/T)
+	var/obj/effect/overlay/temp/cult/sparks/S = PoolOrNew(/obj/effect/overlay/temp/cult/sparks, T)
+	S.anchored = FALSE
+	S.throw_at_fast(src,10,1)
+	var/obj/effect/overlay/temp/bloodsplatter/BS = PoolOrNew(/obj/effect/overlay/temp/bloodsplatter, list(T, get_dir(src, T)))
+	BS.anchored = FALSE
+	BS.throw_at_fast(src,10,1)
+
+/mob/living/simple_animal/hostile/hades/proc/Transfer(var/mob/living/taken, var/turf/transferTarget)
+	if(transferTarget)
+		playsound(get_turf(taken), 'sound/magic/Ethereal_Enter.ogg', 50, 1, -1)
+		var/atom/movable/overlay/hadesFlick/animation = new /atom/movable/overlay/hadesFlick(get_turf(taken))

```

Comment:
Use an `/obj/effect/overlay/temp` subtype, please, this is copying a lot of the code those use.
(see https://github.com/tgstation/tgstation/blob/master/code/game/objects/effects/overlays.dm for relevant code)